types/node-pushnotifications/package.json

```diff
@@ -7,7 +7,8 @@
     ],
     "dependencies": {
         "@types/node": "*",
-        "@types/web-push": "*"
+        "@types/web-push": "*",
+        "firebase-admin": "^11.5.0"
```

Comment:
This seems like a mistake

Reply:
I copied this version from another package which uses firebase-admin types